buildscripts/BuildScripts.csproj

```diff
@@ -1,10 +1,34 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Ah, that looks fine as is with the current name since it is more than just dependencies. My view from the file system (i.e. changes in this pull request) just couldn't see how it would look in VS since I wasn't thinking about files, I was more thinking this was some hidden thing.